extractor/src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInName/Examples/Example2.java

```diff
@@ -0,0 +1,39 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AbbreviationAsWordInName">
+      <property name="ignoreStatic" value="false"/>
+      <property name="ignoreOverriddenMethods" value="false"/>
+    </module>
+  </module>
+</module>
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abbreviationaswordinname;
+
+import java.util.HashSet;
+import java.util.Set;
+
+// xdoc section -- start
+class Example2 extends SuperClass { // OK, camel case
+    int CURRENT_COUNTER; // violation 'no more than '4' consecutive capital letters'
```

Comment:
Please make class `{}`

Reply:
Done

Reply:
I have also refactored the test case outside MainTest.java. Since we are testing all the examples of a specific check, refactored it to a specific folder.